content/applications/hr/employees/new_employee.rst

```diff
@@ -100,82 +82,142 @@ entry and create another line.
+  specific location details, such as a floor or building. If a new work location is needed, add the
+  location by typing it in the field.
+- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
   :guilabel:`Time Off`, :guilabel:`Expenses`, and :guilabel:`Timesheets` for the employee. The
   :guilabel:`External Link` button opens a form with the approver's :guilabel:`Name`,
   :guilabel:`Email Address`, :guilabel:`Company`, :guilabel:`Phone`, and :guilabel:`Mobile` fields.
   These can be modified, if needed. Click :guilabel:`Save` after making any edits.
-- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` (required) and :guilabel:`Timezone` for
-  the employee. The :guilabel:`External Link` button opens up a detailed view of the specific daily
-  working hours. Working hours can be created, modified, or deleted here. Global time off (such as
-  holidays) can be entered in the :guilabel:`Global Time Off` tab. Click :guilabel:`Add a line` to
-  add a new global time off.
-- :guilabel:`Planning`: Click on a planning role from the drop-down menu for both the
-  :guilabel:`Default Planning Role` and the :guilabel:`Planning Roles` fields to add a role. There
-  is no limit to the amount of :guilabel:`Planning Roles` that can be selected for an employee, but
-  there can only be one :guilabel:`Default Planning Role`. The default is the *typical* role that
-  the employee performs, where the :guilabel:`Planning Roles` are *all* the specific roles the
-  employee is able to perform.
+- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` and :guilabel:`Timezone` (both
+  required) for the employee. The :guilabel:`External Link` button opens up a detailed view of the
+  specific daily working hours. Working hours can be modified or deleted here. Click
+  :guilabel:`Save` to save any changes.
+- :guilabel:`Planning`: The :guilabel:`Planning` section affects the *Planning* app, and will only
+  appear if the *Planning* app is installed. Click on a planning role from the drop-down menu for
+  both the :guilabel:`Default Planning Role` and the :guilabel:`Planning Roles` fields to add a
+  role. There is no limit to the amount of :guilabel:`Planning Roles` that can be selected for an
+  employee, but there can only be one :guilabel:`Default Planning Role`. The default is the
+  *typical* role that the employee performs, where the :guilabel:`Planning Roles` are *all* the
+  specific roles the employee is able to perform.
 
   .. image:: new_employee/work-info.png
      :align: center
      :alt: Add the work information to the Work Information tab.
 
-Private information tab
+.. tip::
+   The users that appear in the drop-down menu for the :guilabel:`Approvers` section must have
+   *Administrator* rights set for the corresponding human resources role. To check who has these
+   rights, go to :menuselection:`Settings app --> Users --> → Manage Users`. Click on an employee,
```

Comment:
Not sure if it was intentional to leave the → in Manage Users, but it formats weird and I think it is more readable without it 
![Screenshot 2023-10-19 at 5 40 31 PM](https://github.com/odoo/documentation/assets/36018073/7942a535-fd96-4853-ba74-4d4e09f0dae0)


```suggestion
   rights, go to :menuselection:`Settings app --> Users --> Manage Users`. Click on an employee,
```